src/Instrumentation/PDO/src/PDOInstrumentation.php

```diff
@@ -121,6 +125,18 @@ public static function register(): void
                 $span->setAttributes($attributes);
 
                 Context::storage()->attach($span->storeInContext($parent));
+                if (self::isSqlCommenterEnabled() && $sqlStatement !== 'undefined') {
+                    $sqlStatement = self::appendSqlComments($sqlStatement, true);
+                    $span->setAttributes([
```

Comment:
I think it's redundant to include the sql comments in the statement, that info is already available in the trace. Does the spec suggest to do this? (comment applies to several similar bits of code)

Reply:
Yes, there is no spec / requirement to update `db.query.text`. 
As the instrumentation library changed the sql statement and the database is actually going to execute the modified statement, so updating `TraceAttributes::DB_QUERY_TEXT` to reflect the truth and it helps diagnose too.

Reply:
Tracking the actual (mutated) statement that the DB executed does sound reasonable, but I do share the concern that it makes the value high cardinality. Can we make it opt-in?

Reply:
Sure. Done!